frontend/app_flowy/lib/plugins/grid/presentation/widgets/header/field_cell_action_sheet.dart

```diff
@@ -92,18 +93,30 @@ class _EditFieldButton extends StatelessWidget {
 
 class _FieldOperationList extends StatelessWidget {
   final GridFieldCellContext fieldInfo;
-  final double cellWidth = 100;
-
   const _FieldOperationList(this.fieldInfo, {Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return SizedBox(
-      width: cellWidth * 2,
-      child: Wrap(
-        children: buildCells(),
+    final cells = buildCells();
+    return Column(children: [
+      Flex(
+        direction: Axis.horizontal,
+        children: [
+          cells[0],
+          HSpace(GridSize.typeOptionSeparatorHeight),
+          cells[1],
+        ],
       ),
-    );
+      VSpace(GridSize.typeOptionSeparatorHeight),
+      Flex(
+        direction: Axis.horizontal,
+        children: [
+          cells[2],
```

Comment:
I think hardcoding the cell index would be a problem unless there is a compile-time check.

Reply:
Can we move away from List completely since we're no longer using Wrap/View?

Reply:
Do you mean  using Column instead of Wrap?

Reply:
Yes, by using a couple of Wraps in a Column. I've added a commit to not use List